imagecraft/models/project.py

```diff
@@ -94,6 +102,7 @@ class Project(BaseProject):
     base: BaseT  # type: ignore[reportIncompatibleVariableOverride]
     build_base: BuildBaseT  # type: ignore[reportIncompatibleVariableOverride]
     platforms: dict[str, Platform] | None = None  # type: ignore[assignment, reportIncompatibleVariableOverride]
+    volumes: SingleEntryDict[VolumeName, Volume]
```

Comment:
a SingleEntryDict is a dict with a single key; the spec says that `volumes` can have multiple items, is this here on purpose?

Reply:
The original Volumes spec says that, but the subset of this spec that is defined in the specification you reviewed is stating that a single entry is allowed (this will change in the future but for now we support a single one).